src/elements/AnnotationBlock.ts

```diff
@@ -84,6 +84,24 @@ class AnnotationBlock extends HTMLDivElement {
         }
     }
 
+    /**
+     * Utility function to encode HTML into entities for use in TinyMCE.
+     *
+     * @param {string} content The HTML content to be encoded.
+     * @returns {string} Content with encoded HTML entities.
+     */
+    encodeHTML(content: string): string {
```

Comment:
oh, interesting that this is needed.

any implications for saving annotations? do we need to convert back? (or not, since we're getting the content from tiny?)

probably good to add a simple unit test for this new method

Reply:
> (or not, since we're getting the content from tiny?)

That's right, Tiny formats it correctly with the `getContent` method, which is what we use to save the annotation.